index.html

```diff
@@ -1410,6 +1421,11 @@
           will be <a>JSON-serialized</a>.
         </dd>
       </dl>
+      <p class="note">
```

Comment:
I removed the latter of the 2 instances (leaving the first one earlier in the document).